checks/namespace.go

```diff
@@ -143,7 +142,7 @@ func (c *NamespaceChecker) Check(extConfig external.Check) *pkg.CheckResult {
 		return unexpectedErrorf(check, err, "unable to create namespace")
 	}
 	defer func() {
-		c.Cleanup(ns)
+		c.Cleanup(ns) // nolint: errcheck
```

Comment:
Yes... so the specified type of lint will be ignored for this line

Reply:
I understand. What I'm wondering is if this change is so we can add the linter or if we actually don't want to errcheck here.

Reply:
Its added so we don't check for error in here.

Reply:
Because we don't want to know if there is an error?

Reply:
Normally in defer functions/calls we won't be able to do much when error occurs (as we won't be able to return and use the value) So normally I have seen that we ignore lint checks in these cases, but if there is a better way do let me know :)

Reply:
They can still be logged, right?

Otherwise one can use a callbacks to handle events at the end of async calls.  Check out this pattern in JS (because it's quick for me): https://codesandbox.io/s/mutable-tree-v5ron?file=/src/index.js:0-348

Usually on front end I'd have a function we can pipe stuff like this to and then decide what to do with it. Not sure what the go way is.

We probably would need to take a more global decisions. I don't like swallowing errors though.

Reply:
@BrendanGalloway, views?

Reply:
I am not sure how to handle events at the end of async calls in golang, but I can log. Unless you guys know a better approach